res/menu/qr_show.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android" xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item android:title="@string/menu_share"
+          android:id="@+id/share"
+          android:icon="@drawable/ic_share_white_24dp"
+          app:showAsAction="ifRoom"/>
+
+    <item android:title="@string/menu_copy_link_to_clipboard"
```

Comment:
I used "copy link" instead of "copy" to avoid assumptions that the image would be copied instead the QR data, technically saying the QR content is a link is not incorrect, right? do you think a play "copy" is better?

Reply:
i think, "Copy to Clipboard" is better here, it is neutral and does not say anything about the content - in contrast to "copy link", which can be assumed as being wrong as the copied item is a "link" only technical-wise - the user may expect sth. as "https://..." as a link.

also, desktop uses the neutral form (just "Copy" there), same for ios (if we decide to revive https://github.com/deltachat/deltachat-ios/pull (partly))

Reply:
done